spec/features/tracks_spec.rb

```diff
@@ -39,7 +39,7 @@
       within('table#tracks') do
         expect(page.has_content?(track.name)).to be false
         expect(page.has_content?(track.description)).to be false
-        expect(page.assert_selector('tr', count: 1)).to be true
+        expect(page.has_content?('No data available in table')).to eq true
```

Comment:
Also the feature test is very important to test if this is properly working for users that submit a request (non admin), but this is only testing it for organizers.